src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2294,7 +2310,22 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                         continue;
                                     }
 
-                                    if (!newSymbol.IsImplicitlyDeclared)
+                                    if (IsRecordPrimaryConstructorProperty(edit.OldNode))
+                                    {
+                                        // When deleting a property of a record that represents a primary constructor parameter we need to mark each
+                                        // part of the property as processed so that they don't become delete edits. This isn't an issue for normal
+                                        // properties because adding them is a rude edit so it never gets this far.
```

Comment:
"normal properties" you mean normal properties in records?

Reply:
This should have said deleting. Updated.